tests/pos/tracked.scala

```diff
@@ -0,0 +1,6 @@
+import scala.language.experimental.modularity
```

Comment:
Would be good to add some more tests that exercise the precise conditions that infer `tracked`.